packages/pn-commons/src/utils/apikeys.utility.tsx

```diff
@@ -0,0 +1,141 @@
+import { Box } from '@mui/material';
```

Comment:
@NicolaGiornetta non ha senso che questa utility si trovi in PN commons. Le apiKey hanno scope solo sulle PA. Spostare nel package di riferimento